options.html

```diff
@@ -302,14 +304,12 @@ <h3>Version 4.0.0<span class="changelog-date"> - 8th of February 2020</span></h3
 
 	<div id="tabAudioModal" class="modal">
 		<div class="modal-content">
-			<p class="modal-title">Note!</p>
-			<p>The "Tab Audio Behaviour" option requires additional permissions to function, specifically the "Tabs" permission. Due to a quirk in Chromium, this permission is defined
-				as "Read your browsing history". This permission does NOT give the extension permission to view your actual browsing history, but allows it to see what tabs you currently
-				have open, which we use ONLY to check to see if any tabs are playing audio. If you are okay with that, press "Allow" on the next dialogue.</p>
-			<a target="_blank" href="https://developer.chrome.com/extensions/permission_warnings#view_warnings">For more info, refer to the Chrome Developer Docs.</a>
+			<p id="modal-title"></p>
+			<p id="modal-text"></p>
+			<a id="modal-link" target="_blank"></a>
 			<br />
 			<br />
-			<button id="tabAudioModalDismiss">Understood!</button>
+			<button id="tabAudioModalDismiss"></button>
 		</div>
 	</div>
```

Comment:
Seeing as this is not used anywhere else, shouldn't we remove it completely?

Reply:
Perhaps, but I'd say we keep it so we can have custom popups if we ever need them for anything else. No real harm tbh.

Reply:
No harm at all, but I'd still suggest to remove it so as not to confuse potential contributors. We can always go back to it through Git.

Reply:
Eh, I don't think it'd confuse anyone. It's just a template that's unused.